src/utils.ts

```diff
@@ -554,6 +561,40 @@ export class Utils {
     (target || e.target).dispatchEvent(simulatedEvent);
   }
 
+  /**
+   * defines an element that is used to get the offset and scale from grid transforms
+   * has to be hooked to a helper
+   * should be called once
+  */
+  public static createTransformReferenceElement(): HTMLElement {
+    const transformReference = document.createElement('div');
+    Utils.addElStyles(transformReference, {
+      opacity: '0',
+      position: 'fixed',
+      top: 0 + 'px',
+      left: 0 + 'px',
+      width: '1px',
+      height: '1px',
+      zIndex: '-999999',
+    });
+    return transformReference;
+  }
+
+  /**
+   * can be used after setting the reference element from createTransformReferenceElement
+  */
+  public static getValuesFromTransformedElement(transformReference: HTMLElement, helper: HTMLElement): DragTransform {

```

Comment:
should called 'parent' (where the xform child is added)

Reply:
this one has been changed